docs/pages/Configuration.md

```diff
@@ -25,97 +27,157 @@ A project must define a specification version (`specVersion`), to which its conf
 
 In addition, a project must define a `type`. This can be either `application`, `library`, `theme-library` (since Specification Version 1.1) or `module`. The type defines the default path mappings and build tasks. See [UI5 Builder: Types](./Builder.md#types) for details.
 
-````yaml
-specVersion: "2.2"
-type: library
-````
+!!! example
+
+    === "application"
+
+        ````yaml
+        specVersion: "2.2"
+        type: application
+        ````
+
+    === "library"
+
+        ````yaml
+        specVersion: "2.2"
+        type: library
+        ````
+
+    === "theme-library"
+
+        ````yaml
+        specVersion: "2.2"
+        type: theme-library
+        ````
+
+    === "module"
+
+        ````yaml
+        specVersion: "2.2"
+        type: module
+        ````
 
 ### Metadata
+
+!!! example
+    ````yaml
+    metadata:
+      name: my.company.project
+      copyright: |-
+       My Project
+        * (c) Copyright 2009-${currentYear} My Company
+        * Licensed under the XYZ License, Version n - see LICENSE.txt.
+    ````
+
+#### name
+
 A project must have a `name` and might define a `copyright` string.
 
 In the UI5 Tooling, a project is typically identified by the configured `name`. It must be unique and should ideally follow a namespace scheme like `company.businessarea.project`.
 
+#### copyright
+
 A given `copyright` string will be used to fill placeholders like `${copyright}` and `@copyright@` in a projects source code. `|-` is a way to define a multi line string in YAML. For details, please check the [YAML Specification](https://yaml.org/spec/1.2/spec.html#id2794534).  
 Inside the copyright string, you can use the placeholder `${currentYear}` which will be replaced with the current year.
 
-In case your project is deprecated you may also define a property `deprecated: true`. In projects that have a direct dependency to your project, the UI5 Tooling will then display a deprecation warning.
+#### deprecated
 
-````yaml
-metadata:
-  name: my.cool.project
-  copyright: |-
-   My Cool Project
-    * (c) Copyright 2009-${currentYear} My Company
-    * Licensed under the Apache License, Version 2.0 - see LICENSE.txt.
-````
+In case your project is deprecated you may also define a property `deprecated: true`. In projects that have a direct dependency to your project, the UI5 Tooling will then display a deprecation warning.
```

Comment:
```suggestion
In case your project is deprecated you may also define a property `deprecated: true`. In projects that have a direct dependency to your project, UI5 Tooling will then display a deprecation warning.
```